.github/workflows/manifold.yml

```diff
@@ -10,36 +10,109 @@ on:
   pull_request:
     branches: [ master ]
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
+    strategy:
+      matrix:
+        backend: [CPP, OMP, CUDA]
     runs-on: ubuntu-20.04
     container:
       image: docker://nvidia/cuda:11.6.0-devel-ubuntu20.04
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    - uses: actions/checkout@v2
-    - uses: jwlawson/actions-setup-cmake@v1.12
-    
     - name: Install dependencies
       run: |
         apt-get -y update
-        DEBIAN_FRONTEND=noninteractive apt install -y libglm-dev libassimp-dev libcgal-dev
-    - name: Build CUDA
-      run: |
-        mkdir buildCUDA
-        cd buildCUDA
-        cmake -DCMAKE_BUILD_TYPE=Release .. && make
-    - name: Build CPP
+        DEBIAN_FRONTEND=noninteractive apt install -y libomp-dev git
```

Comment:
Shall we go ahead and install assimp for these builds instead of using the submodule to keep our build times down, like you suggested?

Reply:
Just tried using `libassimp-dev`. It seems that the version is a bit outdated and does not contain `assimp/GltfMaterial.h`. I guess we can just stick to the submodule one for now. As the CI jobs are parallelized, the CI build takes about 20 minutes instead of about an hour previously, it should be a bit more acceptable.